tendrl/commons/message.py

```diff
@@ -153,7 +153,7 @@ def serialize_message(obj):
         serial = obj.isoformat()
         return serial
     elif isinstance(obj, Exception):
-        return yaml.safe_dump(obj)
+        return yaml.safe_dump(obj.message)
```

Comment:
@GowthamShanmugam Also it fails with the error "ruamel.yaml.representer.RepresenterError: cannot represent an object: Exception caught" as in my previous comment. That's why I created a python exception object and tried to yaml.safe_dump the exception object and it fails with same error as in my previous comment.